docker/Dockerfile

```diff
@@ -19,7 +19,7 @@ WORKDIR /celestia_zkevm_ibc_demo
 
 RUN uname -a &&\
     CGO_ENABLED=${CGO_ENABLED} GOOS=${TARGETOS} GOARCH=${TARGETARCH} \
-    make build
+    make build-simapp
```

Comment:
this was the fix. `make build` doesn't do anything b/c it was recently renamed to `make build-simapp`